ci-operator/step-registry/upi/install/openstack/inventory/upi-install-openstack-inventory-commands.sh

```diff
@@ -42,6 +42,8 @@ if [[ "$OS_NETWORKING_TYPE" == 'null' ]]; then
 	errexit=1
 fi
 
```

Comment:
@gryf [This](https://github.com/shiftstack/release/commit/f4689e0ffd074fa518bb8758737f8efe87e20fbf#diff-c31046ac9cec0ba720eb511d1a909eb5564cbdee6492d6d7f30dc3ed9b02ea52R17-R22) is a requirement since we're running on a cloud with Proxy, otherwise we wouldn't be able to reach the API and bootstrap would fail. Result were much better when running with that.

Can you also change the name of the jobs to e2e-openstack-dualstack-upi and [modify the flavor of worker](https://github.com/shiftstack/release/commit/f4689e0ffd074fa518bb8758737f8efe87e20fbf#diff-609fb532596692d51bf26a55c8a4b9f0965c3260e3293501ef5fe4db6b2e01f3R13) to be the same of master?